bundles/org.openhab.binding.electroluxair/src/main/java/org/openhab/binding/electroluxair/internal/handler/ElectroluxAirHandler.java

```diff
@@ -0,0 +1,226 @@
+                        Units.PERCENT);
+            case CHANNEL_TVOC:
+                return new QuantityType<Dimensionless>(dto.getTwin().getProperties().getReported().gettVOC(),
+                        Units.PARTS_PER_BILLION);
+            case CHANNEL_PM1:
+                return new QuantityType<Density>(dto.getTwin().getProperties().getReported().getpM1(),
+                        Units.MICROGRAM_PER_CUBICMETRE);
+            case CHANNEL_PM25:
+                return new QuantityType<Density>(dto.getTwin().getProperties().getReported().getpM25(),
+                        Units.MICROGRAM_PER_CUBICMETRE);
+            case CHANNEL_PM10:
+                return new QuantityType<Density>(dto.getTwin().getProperties().getReported().getpM10(),
+                        Units.MICROGRAM_PER_CUBICMETRE);
+            case CHANNEL_CO2:
+                return new QuantityType<Dimensionless>(dto.getTwin().getProperties().getReported().getcO2(),
+                        Units.PARTS_PER_MILLION);
+            case CHANNEL_FAN_SPEED:
+                return new StringType(Integer.toString(dto.getTwin().getProperties().getReported().getFanspeed()));
+            case CHANNEL_FILTER_LIFE:
+                return new QuantityType<Dimensionless>(dto.getTwin().getProperties().getReported().getFilterLife(),
+                        Units.PERCENT);
+            case CHANNEL_IONIZER:
+                return OnOffType.from(dto.getTwin().getProperties().getReported().ionizer);
+            case CHANNEL_WORK_MODE:
+                return new StringType(dto.getTwin().getProperties().getReported().workmode);
+            case CHANNEL_DOOR_OPEN:
+                return dto.getTwin().getProperties().getReported().doorOpen ? OpenClosedType.OPEN
+                        : OpenClosedType.CLOSED;
+
+        }
+        return UnDefType.UNDEF;
+    }
+
+    private Map<String, String> refreshProperties() {
+        Map<String, String> properties = new HashMap<>();
+        Bridge bridge = getBridge();
+        if (bridge != null) {
+            ElectroluxAirBridgeHandler bridgeHandler = (ElectroluxAirBridgeHandler) bridge.getHandler();
+            if (bridgeHandler != null) {
+                ElectroluxPureA9DTO dto = bridgeHandler.getElectroluxAirThings().get(config.getDeviceId());
+                if (dto != null) {
+                    properties.put(PROPERTY_BRAND, dto.getApplicancesInfo().brand);
+                    properties.put(PROPERTY_COLOUR, dto.getApplicancesInfo().colour);
+                    properties.put(PROPERTY_DEVICE, dto.getApplicancesInfo().device);
+                    properties.put(PROPERTY_MODEL, dto.getApplicancesInfo().model);
+                    properties.put(PROPERTY_SERIAL_NUMBER, dto.getApplicancesInfo().serialNumber);
+                    properties.put(PROPERTY_FW_VERSION, dto.getTwin().getProperties().getReported().frmVerNIU);
```

Comment:
There are predefined properties you could use.
```suggestion
                    properties.put(Thing.PROPERTY_VENDOR, dto.getApplicancesInfo().brand);
                    properties.put(PROPERTY_COLOUR, dto.getApplicancesInfo().colour);
                    properties.put(PROPERTY_DEVICE, dto.getApplicancesInfo().device);
                    properties.put(Thing.PROPERTY_MODEL_ID, dto.getApplicancesInfo().model);
                    properties.put(Thing.PROPERTY_SERIAL_NUMBER, dto.getApplicancesInfo().serialNumber);
                    properties.put(Thing.PROPERTY_FIRMWARE_VERSION,
                            dto.getTwin().getProperties().getReported().frmVerNIU);
```